content/news/core-monthly/2025/core-monthly-2025-01-01-2025-01-31.md

```diff
@@ -0,0 +1,300 @@
+* [#1112](https://github.com/PrestaShop/autoupgrade/pull/1112): [NEW UI] Update send error message dialog. Thank you, [@ga-devfront](https://github.com/ga-devfront)
+* [#1111](https://github.com/PrestaShop/autoupgrade/pull/1111): Refactor: uniform class instantiations. Thank you, [@ga-devfront](https://github.com/ga-devfront)
+* [#1110](https://github.com/PrestaShop/autoupgrade/pull/1110): [NEW UI] Restore page. Thank you, [@ga-devfront](https://github.com/ga-devfront)
+* [#1109](https://github.com/PrestaShop/autoupgrade/pull/1109): Refactor Codebase to Use Symfony Filesystem Instead of Native PHP Functions. Thank you, [@M0rgan01](https://github.com/M0rgan01)
+* [#1108](https://github.com/PrestaShop/autoupgrade/pull/1108): [NEW UI] Add restore choice dialogs. Thank you, [@ga-devfront](https://github.com/ga-devfront)
+* [#1107](https://github.com/PrestaShop/autoupgrade/pull/1107): Introduce Error 404 page. Thank you, [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#1106](https://github.com/PrestaShop/autoupgrade/pull/1106): Translation catalog update. Thank you, [@github-actions](https://github.com/github-actions)
+* [#1105](https://github.com/PrestaShop/autoupgrade/pull/1105): [NEW UI] use config for restore state. Thank you, [@ga-devfront](https://github.com/ga-devfront)
+* [#1104](https://github.com/PrestaShop/autoupgrade/pull/1104): Bump to v7.0.0. Thank you, [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#1103](https://github.com/PrestaShop/autoupgrade/pull/1103): [LOG] Improve logs wording, add classname in log files. Thank you, [@M0rgan01](https://github.com/M0rgan01)
+* [#1102](https://github.com/PrestaShop/autoupgrade/pull/1102): [DOC] Add configuration parameters section in readme. Thank you, [@M0rgan01](https://github.com/M0rgan01)
+* [#1101](https://github.com/PrestaShop/autoupgrade/pull/1101): [NEW UI] Post restore page. Thank you, [@ga-devfront](https://github.com/ga-devfront)
+* [#1099](https://github.com/PrestaShop/autoupgrade/pull/1099): [NEW UI] restore page backup choice. Thank you, [@ga-devfront](https://github.com/ga-devfront)
+* [#1098](https://github.com/PrestaShop/autoupgrade/pull/1098): Improve archive download. Thank you, [@M0rgan01](https://github.com/M0rgan01)
+* [#1097](https://github.com/PrestaShop/autoupgrade/pull/1097): Translation catalog update. Thank you, [@github-actions](https://github.com/github-actions)
+* [#1095](https://github.com/PrestaShop/autoupgrade/pull/1095): [CLI] Add stack trace in logs when exception. Thank you, [@M0rgan01](https://github.com/M0rgan01)
+* [#1088](https://github.com/PrestaShop/autoupgrade/pull/1088): UI tests - Add first scénario to upgrade from module interface. Thank you, [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+### Cash on delivery module
+* [#61](https://github.com/PrestaShop/ps_cashondelivery/pull/61): Functional Tests : Added dependabot & Bump `@prestashop-core/ui-testing`. Thank you, [@Progi1984](https://github.com/Progi1984)
```

Comment:
```suggestion
* [#61](https://github.com/PrestaShop/ps_cashondelivery/pull/61): Functional Tests: Added dependabot & Bump `@prestashop-core/ui-testing`. Thank you, [@Progi1984](https://github.com/Progi1984)
```

Reply:
@coffrep, thanks, Patrick. In the future, there is no need to do the proofreading of the generated content from this article